File: excel-sheet-column-number.cs
public class Solution {
    public int TitleToNumber(string s) {
        int colNum = 0;
        for(int i=s.Length-1,j=0;i>=0;i--,j++)
            colNum += (s[i]-'A'+1) * (int)Math.Pow(26,j);
        return colNum;
    }
}

File: Excel Sheet Column Number.java
class Solution {
    public int titleToNumber(String s) {
        char[] ch = s.toCharArray();
        int ans = 0;
        for(int i=0;i<ch.length;i++)
        {
            ans *= 26;
            ans += ch[i] - 'A' + 1;
        }
        return ans;
    }
}
